server/moodle/mod/livequiz/classes/models/question.php

```diff
@@ -200,4 +201,52 @@ public function add_answer(answer $answer): void {
     private function set_id($id): void {
         $this->id = $id;
     }
+
+    /**
+     * Getter for question hasmultiplecorrectanswers
+     * @return bool
+     */
+    public function get_hasmultiplecorrectanswers(): bool {
+        // This is a simple check to see if the question has multiple correct answers.
+        $numcorrect = 0;
+        $hasmultipleanswers = false;
+        foreach ($this->answers as $answer) {
+            if ($answer->get_correct()) {
+                $numcorrect++;
+            }
+        }
+        if ($numcorrect > 1) {
+            $hasmultipleanswers = true;
+        }
+        return $hasmultipleanswers;
```

Comment:
For at undgå ønedvendige iterationer. I må selv om i vil bruge det

```suggestion
        // This is a simple check to see if the question has multiple correct answers.
       $numcorrect = 0;

    foreach ($this->answers as $answer) {
        if ($answer->get_correct()) {
            $numcorrect++;
            if ($numcorrect > 1) {
                return true;
            }
        }
    }

    return false;
```